fdbclient/NativeAPI.actor.cpp

```diff
@@ -7535,6 +7540,9 @@ Future<Void> Transaction::onError(Error const& e) {
 			++trState->cx->transactionsProcessBehind;
 		else if (e.code() == error_code_batch_transaction_throttled || e.code() == error_code_tag_throttled) {
 			++trState->cx->transactionsThrottled;
+		} else if (e.code() == error_code_proxy_tag_throttled) {
+			++trState->cx->transactionsThrottled;
+			trState->proxyTagThrottledDuration += CLIENT_KNOBS->PROXY_MAX_TAG_THROTTLE_DURATION;
```

Comment:
Where is this client knob PROXY_MAX_TAG_THROTTLE_DURATION being used?

Reply:
It is used on the GRV proxy to determine when to reject transactions that have been queued too long.

Reply:
I think what confused me is that PROXY_MAX_TAG_THROTTLE_DURATION is a client knob. Does it automatically get transferred to the server somehow?